src/mono/wasm/runtime/invoke-cs.ts

```diff
@@ -97,23 +101,28 @@ export function mono_wasm_bind_cs_function(fully_qualified_name: MonoStringRef,
 function bind_fn_0V(closure: BindingClosure) {
     const method = closure.method;
     (<any>closure) = null;
+    const fqn = closure.fqn;
```

Comment:
This also looks wrong.  `const fqn = clossure.fqn` should be before setting closure to `null`?